src/platyPS/platyPS.psm1

```diff
@@ -787,79 +789,101 @@ function New-ExternalHelp
 
     end 
     { 
-        # write verbose output and filter out files based on applicable tag
-        $MarkdownFiles | ForEach-Object {
+       # Tracks all warnings and errors 
+       $warningsAndErrors = New-Object System.Collections.Generic.List[System.Object]
+
+       try {
```

Comment:
Not sure is it just github diff problems, but as far as I can tell you just added a try-catch. Did you replace tabs to space at the same time? The diff looks confusing.

Reply:
Let me see what I can do to fix.

Reply:
I cloned a fresh copy of repo.  The original file and mine both use spaces instead of tabs and a size of 4.  I added a Try, Catch, Finally block so I could log any errors that happened.  That would indent everything over some.